salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -1,6 +1,8 @@
 {%- set node_name = pillar.orchestrate.node_name %}
 {%- set version = pillar.metalk8s.nodes[node_name].version %}
 
+{%- set skip_roles = pillar.metalk8s.nodes[node_name].get('skip_roles', []) %}
```

Comment:
Maybe source it from `pillar.orchestrate`?

Reply:
Hmm no, it's not really intuitif that mean if someone want to skip etcd from CLI directly when running highstate he will need to change the one in `metalk8s.nodes` and if he want to skip using `deploy_node` he can only use `orchestrate` one.
Even if it's not a feature we should use really often for me it's better to have only one place to put it in the pillar